Sources/MongoSwift/WriteConcern.swift

```diff
@@ -113,6 +116,13 @@ public struct WriteConcern: Codable {
         }
     }
 
+    /// Initializes a new `WriteConcern` without throwing any errors.
+    fileprivate init(journal: Bool?, w: W, wtimeoutMS: Int?) {
```

Comment:
rather than using this and perhaps silently making an invalid write concern, we can just re-use the other one with `try!`. Because it's a static variable, if this ever fails we'll know about it immediately.

Reply:
I agree that would be cleaner, it does cause a linter error however (no `force_try`) I added a comment to avoid the error for now, do we wanna add skipping this rule globally?

Reply:
No, in most cases we never want to force try because it would crash a user application if it failed. This is okay here because it's in static space, so if our tests pass then we know it isn't failing.